docs/migrating_to_swagger_20.rst

```diff
@@ -3,12 +3,13 @@ Migrating to Swagger 2.0
 
 So you're using pyramid_swagger with Swagger 1.2 and now it is time to upgrade to Swagger 2.0.
 
-Just set the version of Swagger to support via configuration like so:
+Just set the version of Swagger to support via configuration. Also, set ``use_models`` to ``false`` so that your existing Swagger 1.2 views will continue to received models in incoming requests as dicts instead of Python classes.
```

Comment:
An example may better illustrate the behavior here.

With ```use_models=False```:

```python
@view_config(route_name='add.user)
def add_user(request):
    user = request.swagger_data['user']
    assert isinstance(user, dict)
    firstname = user['first_name']
    
    ...
```

With ```use_models=True```:
```python
@view_config(route_name='add.user)
def add_user(request):
    user = request.swagger_data['user']
    assert isinstance(user, bravado_core.models.User)
    firstname = user.first_name
    ...
```

No data bound to the request is changed. Only the representation of ```#/definitions``` in ```request.swagger_data``` is affected.

I'll make the suggested change to use "existing Pyramid views".